docs/v1/P4Runtime-Spec.adoc

```diff
@@ -1757,31 +1573,35 @@ Both `Meter` and `DirectMeter` messages share the following fields:
   this meter extern instance. The `MeterSpec` message is used to describe the
   meter unit and the meter type. The meter unit can be any of the
   `MeterSpec.Unit` enum values:
-    * `UNSPECIFIED`: reserved value.
-    * `BYTES`, which signifies that this meter can be configured with rates
+    ** `UNSPECIFIED`: reserved value.
+    ** `BYTES`, which signifies that this meter can be configured with rates
       expressed in bytes/second.
-    * `PACKETS`, for rates expressed in packets/second.
-
-  The meter type can be any of the `MeterSpec.Type` enum values:
+    ** `PACKETS`, for rates expressed in packets/second.
+    +
+    The meter type can be any of the `MeterSpec.Type` enum values:
+    +
     * `TWO_RATE_THREE_COLOR`: This is the *Two Rate Three Color Marker* (trTCM)
-      defined in RFC 2698 [@RFC2698]. This is the standard P4Runtime meter type
+      defined in RFC 2698 cite:[RFC2698]. This is the standard P4Runtime meter type
       and allows meters to use two rates to split packets into three potential
       colors: GREEN, YELLOW, or RED. This mode is the default, but can also be
       set explicitly in a P4 program by adding the `@two_rate_three_color`
       annotation to the meter definition.
       For example, in a V1Model P4 program, we might define a trTCM direct meter
       as follows:
-      ~ Begin P4Example
+      +
+      [source,p4]
```

Comment:
I think `[source,p4]` might need to have no whitespace before it, too, but not sure.